e2e/cypress/support/ui_commands.js

```diff
@@ -91,6 +91,16 @@ Cypress.Commands.add('postMessageReplyInRHS', (message) => {
     postMessageAndWait('#reply_textbox', message);
 });
 
+Cypress.Commands.add('postMessageQuickly', (message) => {
```

Comment:
Please change to `uiPostMessageQuickly` to easily check at a glance that this is done via UI. It's our recent convention for UI commands, although not many were migrated yet. Also, please add types at `ui_commands.d.ts`.  Thanks!